### PR TITLE
Faq Updates

### PR DESCRIPTION
**Pull request by RevolutionTech**

### CODE REVIEW

File: templates/extra/faq.html [hunk 1]
```diff
@@ -1,176 +1,67 @@
-{% extends "base.html" %}
-
-{% block content %}
-    <h2>FAQ</h2>
+{% extends "base.html" %} {% block content %}
```
Comment (RevolutionTech) [inline templates/extra/faq.html]: Content block should begin on its own line, and content should be indented in:
```
{% extends "base.html" %}

{% block content %}
    <h2>FAQ</h2>
    ...
{% endblock %}
```

File: templates/extra/faq.html [hunk 2]
```diff
@@ -1,176 +1,67 @@
...
+    <h5 class="color-blue">What happens if an artist leaves the platform?</h5>
+    <p>
+        The system is designed to protect both the artists and the fans. If an artist decides to leave the platform we highly encourage them to do it in a way that doesn't upset or disappoint their investors. In the past, some artists that were unable to fulfill their campaign goals have refunded the money to investors, while others have not. Investing in music is high risk and we encourage people to only invest what they are willing to lose.
+    </p>
+    <div class="break">
+        <hr>
+    </div>
+    <h5 class="color-blue">How does PerDiem make money?</h5>
+    <p>
+        100% of the money from shares purchased goes to the artists, and 100% of revenue is split between the artist and fans. The only fee for the investor is the credit card processing fee + 12.9% + $0.30.
```
Comment (RevolutionTech) [inline templates/extra/faq.html]: Is the 12.9% correct? That sounds very high.

File: templates/extra/faq.html [hunk 3]
```diff
@@ -1,176 +1,67 @@
...
+    </p>
+    <p>
+        For example: You invest $10 in an artist. The total would be $11.41. $10 goes to the artist, $1 goes to Perdiem, $0.41 goes to the credit card processor.
```
Comment (RevolutionTech) [inline templates/extra/faq.html]:
```suggestion
        For example: You invest $10 in an artist. The total would be $11.41. $10 goes to the artist, $1 goes to PerDiem, $0.41 goes to the credit card processor.
```

File: templates/extra/faq.html (hunk 2 above)
Comment (localastronaut) [reply templates/extra/faq.html]: No, I wrote that line incorrectly. 

I mentioned it on slack a little while ago, but I want to change the $1 fee to 10%. I tried updating it on the backend but ran into some issues. I'll shoot you a message on slack about.

**Pull request merged**